code/datums/actions/items/beserk.dm

```diff
@@ -7,14 +7,27 @@
 	overlay_icon_state = "bg_demon_border"
 
 /datum/action/item_action/berserk_mode/Trigger(trigger_flags)
-	if(istype(target, /obj/item/clothing/head/hooded/berserker))
-		var/obj/item/clothing/head/hooded/berserker/berserk = target
-		if(berserk.berserk_active)
+	. = ..()
+	if(!.)
+		return FALSE
+	var/obj/item/clothing/head/hooded/berserker/berserk = target
+	berserk.berserk_mode(owner)
+	return TRUE
+
+/datum/action/item_action/berserk_mode/IsAvailable(feedback = FALSE)
+	. = ..()
+	if(!.)
+		return FALSE
+	if(!istype(target, /obj/item/clothing/head/hooded/berserker))
+		return FALSE
+
+	var/obj/item/clothing/head/hooded/berserker/berserk = target
+	if(berserk.berserk_active)
+		if(feedback)
```

Comment:
...Because you correctly put the to chat message behind `feedback`, as it should be